go/exocom/main_test.go

```diff
@@ -189,16 +189,18 @@ func FeatureContext(s *godog.Suite) {
 			return fmt.Errorf("Expected to receive a message but got %v", err)
 		}
 		outgoingMessageActivityID = actualMessage.ActivityID
-		payload, ok := actualMessage.Payload.(map[string]interface{})
-		if !ok {
-			return errors.New("Error: cannot parse payload")
-		}
-		delete(payload, "sessionId")
-		delete(payload, "responseTime")
-		delete(payload, "sender")
-		delete(payload, "timestamp")
-		if !reflect.DeepEqual(expectedMessage.Payload, payload) {
-			return fmt.Errorf("Expected payload to equal %s but got %s", expectedMessage.Payload, actualMessage.Payload)
+
+		expectedBytes, err := json.Marshal(expectedMessage.Payload)
+		actualBytes, err := json.Marshal(actualMessage.Payload)
+		actualMessagePayload := structs.Message{}
+		expectedMessagePayload := structs.Message{}
+		err = json.Unmarshal(actualBytes, &actualMessagePayload)
+		err = json.Unmarshal(expectedBytes, &expectedMessagePayload)
+		actualMessagePayload.Sender = expectedMessagePayload.Sender
```

Comment:
Why is this being overwritten? The sender seems like something we should be able to hardcode in the JSON of the feature